stable/latest/docs

```diff
@@ -1 +1 @@
-../0.15/docs
```

Comment:
This is wrong. It should be `../0.16/docs` instead of `..//docs`

Reply:
Yes, which is why I mentioned that something is missing in the release description.